lib/utils/index.js

```diff
@@ -273,6 +273,48 @@ module.exports = {
   },
 
   /**
+   * Gets the property name of a given node.
+   * @param {ASTNode} node - The node to get.
+   * @return {string|null} The property name if static. Otherwise, null.
+   */
+  getStaticPropertyName (node) {
```

Comment:
It's sweet memories, I wrote it at [eslint/lib/ast-utils](https://github.com/eslint/eslint/blame/2874d75ed8decf363006db25aac2d5f8991bd969/lib/ast-utils.js#L878).
I hope `eslint/lib/ast-utils` to separate to another package (though it's an item of ESLint 5 wish list).